docs/fuse.md

```diff
@@ -65,7 +65,7 @@ ipfs daemon --mount
 If you wish to allow other users to use the mount points, use the following:
 
 ```sh
-ipfs config Mounts.FuseAllowOther --bool true
```

Comment:
Ok, so here is a clear case of what i'm talking about:

```
ipfs config Mounts.FuseAllowOther true  # bool
ipfs config Mounts.FuseAllowOther "true"  # bool
ipfs config Mounts.FuseAllowOther '"true"'  # string
```

if i wanted the value `"true"`, i would expect the 2nd line to work.